lesson6Khanh/src/main/java/com/trainingdeveloperpro/androidk001/lesson5/nguyenquockhanh/studentlist/StudentViewHolder.kt

```diff
@@ -0,0 +1,26 @@
+package com.trainingdeveloperpro.androidk001.lesson5.nguyenquockhanh.studentlist
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.ImageButton
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+
+class StudentViewHolder(inflater: LayoutInflater, parent: ViewGroup) : RecyclerView.ViewHolder(inflater.inflate(R.layout.recycler_view, parent, false)){
+    private var userName: TextView? = null
+    private var userAge: TextView? = null
+    private var userTelephone: TextView? = null
```

Comment:
Đoạn này có thể dùng lateinit để đỡ phải gán giá trị null
```suggestion
    private lateinit var userTelephone: TextView
```